.travis.yml

```diff
@@ -2,34 +2,24 @@ dist: bionic
 language: python
 python: 3.7
 
-matrix:
+jobs:
   include:
     - name: "pytype"
       python: 3.6
-      env:
-        - TEST_CMD="./tests/pytype_test.py"
-        - INSTALL="test"
+      install: pip install -r requirements-tests-py3.txt
+      script: ./tests/pytype_test.py
     - name: "mypy (typed-ast)"
-      env:
-        - TEST_CMD="./tests/mypy_test.py"
-        - INSTALL="mypy"
+      python: 3.7
+      install: pip install -U git+git://github.com/python/mypy git+git://github.com/python/typed_ast
+      script: ./tests/mypy_test.py
     - name: "mypy (ast)"
       python: 3.8-dev
-      env:
-        - TEST_CMD="./tests/mypy_test.py"
-        - INSTALL="mypy"
+      install: pip install -U git+git://github.com/python/mypy
```

Comment:
Are you sure `typed_ast` should be removed here? As I understand it, mypy still depends on it even on 3.8.

Reply:
That doesn't seem to be the case, since the build succeeds. At least at runtime it should not need it.

Reply:
Didn't check but it probably installs `typed_ast` from PyPI because it is a dependency of mypy. But the intention was to check with the git `typed_ast`.

Reply:
I just looked at the travis log. It is indeed installing typed-ast from pypi, but I assume that it will not get used by mypy.

Reply:
I asked about it [here](https://github.com/python/mypy/pull/6539#issuecomment-473456639) -- according to the replies, it it still needed in Python 3.8 to parse Python 2 code.

Reply:
Ah, good point. But this shouldn't be a problem for us, since we are only parsing stubs and are not running mypy in Python 2 mode. I think using typed-ast from pypi is fine, but in the end it doesn't matter much, especially considering that typed-ast will probably not be developed any further.

I actually think it might be beneficial to use typed-ast from pypi in the < 3.8 test, but that's a discussion for another PR.